trtools/dumpSTR/tests/test_dumpSTR.py

```diff
@@ -58,18 +60,51 @@ def args(tmpdir):
     args.verbose = False
     return args
 
+@pytest.fixture
+def testDumpSTRdir(vcfdir):
+    return vcfdir + "/dumpSTR_vcfs"
+
 # Test no such file or directory
-def test_WrongFile(args, vcfdir):
-    fname = os.path.join(vcfdir, "test_non_existent.vcf")
+def test_WrongFile(args, testDumpSTRdir):
+    fname = os.path.join(testDumpSTRdir, "test_non_existent.vcf")
     if os.path.exists(fname):
         os.remove(fname)
     args.vcf = fname
     retcode = main(args)
     assert retcode==1
 
+def test_BadPreexistingFields(args, testDumpSTRdir, capsys):
+    fname = os.path.join(testDumpSTRdir, "bad_preexisting_hrun.vcf")
+    args.vcf = fname
+    retcode = main(args)
+    captured = capsys.readouterr()
+    assert "HRUN" in captured.err
+
+    fname = os.path.join(testDumpSTRdir, "bad_preexisting_het_hwep.vcf")
+    args.vcf = fname
+    retcode = main(args)
+    captured = capsys.readouterr()
+    assert "HWEP" in captured.err and "HET" in captured.err
+
+    fname = os.path.join(testDumpSTRdir, "bad_preexisting_filter_ac_refac.vcf")
+    args.vcf = fname
+    retcode = main(args)
+    captured = capsys.readouterr()
+    assert ("FILTER" in captured.err and "AC" in captured.err
+            and "REFAC" in captured.err)
+
+def test_WorrisomePreexistingFilter(args, testDumpSTRdir, capsys):
+    fname = os.path.join(testDumpSTRdir, "worrisome_preexisting_filter.vcf")
+    args.vcf = fname
+    args.min_locus_hwep = 0.5
+    retcode = main(args)
+    assert retcode == 0
+    captured = capsys.readouterr()
+    assert 'HWE0.5' in captured.err
+
 # Test if basic inputs and threshold filters work for each file
-def test_GangSTRFile(args, vcfdir):
-    fname = os.path.join(vcfdir, "test_gangstr.vcf")
+def test_GangSTRFile(args, testDumpSTRdir):
+    fname = os.path.join(testDumpSTRdir, "trio_chr21_gangstr.sorted.vcf.gz")
     args.vcf = fname
```

Comment:
I think the tests under these functions only check if the code works without error. Unless I'm missing something, I think we need to tests under test_filters.py for the actual functionality of each filter that is implemented.